test/async-hooks/test-signalwrap.js

```diff
@@ -22,7 +22,7 @@ assert.strictEqual(typeof signal1.triggerAsyncId, 'number');
 checkInvocations(signal1, { init: 1 }, 'when SIGUSR2 handler is set up');
 
 let count = 0;
-exec('kill -USR2 ' + process.pid);
+exec(`kill -USR2 ${process.pid}`);
```

Comment:
Should use `process.kill(pid, signal)` - https://nodejs.org/api/process.html#process_process_kill_pid_signal